ci/vm/entrypoint.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -xe
+
+OS=$1
+
+install_packages() {
+  if [[ $OS == *"ubuntu"* ]]; then
+    apt-get update &&
+      DEBIAN_FRONTEND=noninteractive apt-get install -yq \
+        automake \
+        dkms \
+        git \
+        libtool &&
+      apt-get clean && rm -rf /var/lib/apt/lists/*
+  elif [[ $OS == *"centos"* ]]; then
+    yum install -y -q centos-release-scl
+    yum install -y -q \
+      automake \
+      devtoolset-8-gcc \
+      devtoolset-8-gcc-c++ \
+      elfutils-libelf-devel \
+      git \
+      libtool \
+      make &&
+      yum clean all
+  fi
+}
+
+xpmem_build() {
+  git clone -q https://github.com/openucx/xpmem.git
```

Comment:
Why do we clone it here? Do we test master or PR branch?

Reply:
That's a good point!
Need to remake this solution.

Reply:
Building XPMEM here as a part of VM preparation. 
This is done to keep the CI wrapper minimalistic and concise.